content/library/advanced-features/connecting-to-data.md

```diff
@@ -12,15 +12,15 @@ Most Streamlit apps need some kind of data or API access to be useful - either r
 - Connecting to data in a Python application is often tedious and annoying.
 - There are specific considerations for connecting to data from streamlit apps, such as caching and secrets management.
 
-**Streamlit provides [`st.experimental_connection()`](/library/api-reference/connections/st.experimental_connection) to more easily connect your Streamlit apps to data and APIs with just a few lines of code**. This page provides a basic example of using the feature and then focuses on advanced usage.
+**Streamlit provides [`st.connection()`](/library/api-reference/connections/st.connection) to more easily connect your Streamlit apps to data and APIs with just a few lines of code**. This page provides a basic example of using the feature and then focuses on advanced usage.
 
 For a comprehensive overview of this feature, check out this video tutorial by Joshua Carroll, Streamlit's Product Manager for Developer Experience. You'll learn about the feature's utility in creating and managing data connections within your apps by using real-world examples.
 
 <YouTube videoId="xQwDfW7UHMo" />
 
 ## Basic usage
 
-For basic startup and usage examples, read up on the relevant [data source tutorial](/knowledge-base/tutorials/databases) or our [blog post introducing st.experimental_connection](https://blog.streamlit.io/introducing-st-experimental_connection/). Streamlit has built-in connections to SQL dialects and Snowflake Snowpark. We also maintain installable connections for [Cloud File Storage](https://github.com/streamlit/files-connection) and [Google Sheets](https://github.com/streamlit/gsheets-connection).
+For basic startup and usage examples, read up on the relevant [data source tutorial](/knowledge-base/tutorials/databases). Streamlit has built-in connections to SQL dialects and Snowflake Snowpark. We also maintain installable connections for [Cloud File Storage](https://github.com/streamlit/files-connection) and [Google Sheets](https://github.com/streamlit/gsheets-connection).
```

Comment:
Update: remove "Snowpark" (so sentence becomes "connections to SQL dialects and Snowflake."

Reply:
Done in [6f5c73b](https://github.com/streamlit/docs/pull/860/commits/6f5c73b9affdee566e3a4d81f05cac6992deb3e5)